compiler/passes/convert-uast.cpp

```diff
@@ -468,7 +468,7 @@ struct Converter {
   }
 
   BlockStmt* convertUseNoLimitations(const uast::Use* node) {
-    auto args = new std::vector<PotentialRename*>();
+    auto args = new std::vector<PotentialRename*>(); // TODO LEAK
```

Comment:
Was this an already existing leak that you found and just leaving a TODO for it, or was this introduced here?

Reply:
Already existing. It popped up with asan before adding detect_leaks=0 but didn't want to forget about it

Reply:
That's a shame that this leaks. It means that either the old compiler code that uses this stuff leaks, or (hopefully), I forgot a delete somewhere. I think it would be ideal if we didn't heap allocate these things at all, but we can't really consider that until we replace the parser.

Reply:
Sure.